src/faebryk/libs/picker/picker.py

```diff
@@ -147,6 +147,17 @@ def mark_no_pick_needed(module: Module):
         )
 
 
+class is_virtual(Module.TraitT):
```

Comment:
This should be a `decless` trait
```suggestion
class is_virtual(Module.TraitT.decless()):
```